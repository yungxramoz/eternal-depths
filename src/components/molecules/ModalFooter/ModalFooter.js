import React from 'react'
import './ModalFooter.css'
import RpgSeparator from '../../templates/RpgSeperator/RpgSeparator'

const ModalFooter = ({ children }) => (
  <div className="modal-footer">
    <RpgSeparator golden={false} />
    {children}
  </div>
)

export default ModalFooter
