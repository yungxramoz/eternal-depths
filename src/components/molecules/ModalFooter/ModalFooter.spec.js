import { render } from '@testing-library/react'
import React from 'react'
import ModalFooter from './ModalFooter'

describe('ModalFooter', () => {
  it('renders correctly', () => {
    render(<ModalFooter>Test Content</ModalFooter>)
    expect(document.body).toMatchSnapshot('ModalFooter')
  })
})
