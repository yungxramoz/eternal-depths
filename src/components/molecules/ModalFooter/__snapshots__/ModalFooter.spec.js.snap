// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalFooter renders correctly: ModalFooter 1`] = `
<body>
  <div>
    <div
      class="modal-footer"
    >
      <hr
        class="separator"
      />
      Test Content
    </div>
  </div>
</body>
`;
